src/constants.py

```diff
@@ -5,6 +5,8 @@
 
 CHECK_READY_NAME = "synapse-ready"
 COMMAND_MIGRATE_CONFIG = "migrate_config"
+SYNAPSE_CONFIG_DIR = "/data"
+SYNAPSE_CONFIG_PATH = SYNAPSE_CONFIG_DIR + "/homeserver.yaml"
```

Comment:
Why not an f-string?

Reply:
Golang bad habit 🤦Thanks for pointing that out!